consensus/parlia/parlia.go

```diff
@@ -881,8 +881,20 @@ func (p *Parlia) Seal(chain consensus.ChainHeaderReader, block *types.Block, res
 		}
 	}
 
-	// Sweet, the protocol permits us to sign the block, wait for our time
-	delay := p.delayForRamanujanFork(snap, header)
+	// BEP-188 allows an in-turn validator to broadcast the mined block earlier
+	// but not earlier than its parent's timestamp after Bohr fork.
+	delay := time.Duration(0)
+	if p.chainConfig.IsBohr(header.Number) && header.Difficulty.Cmp(diffInTurn) == 0 {
+		parent := chain.GetHeader(header.ParentHash, number-1)
+		if parent == nil || parent.Number.Uint64() != number-1 || parent.Hash() != header.ParentHash {
+			return consensus.ErrUnknownAncestor
+		}
+		if parent.Time > uint64(time.Now().Unix()) {
+			delay = time.Until(time.Unix(int64(parent.Time), 0))
```

Comment:
This can only be allowed when the gas limit is almost exhausted, isn't?

Reply:
agree, could add a gasUsed ratio check, such as `gasUsed/gasLimit >= 3/4`